backend/src/wamp.js

```diff
@@ -161,6 +163,10 @@ wampHandlers["get-account-details"] = async ([accountId]) => {
     nearRpc.sendJsonRpc("EXPERIMENTAL_genesis_config", {}),
   ]);
 
+  if (accountInfo === null) {
+    return null;
+  }
```

Comment:
It is easier to handle the missing account information when it returns `null` instead of raising some cryptic error message